### PR TITLE
Add contract state tree to top level tree + fork chain

### PR DESCRIPTION
**Pull request by hanssv**
This should fix the issue seen on the integration network.

### CODE REVIEW

File: apps/aecontract/src/aect_state_tree.erl [hunk 1]
```diff
@@ -34,3 +36,7 @@ insert_contract(_Contract, Tree) ->
     %% PLACEHOLDER
     Tree.
 
+-spec root_hash(tree()) -> {ok, aeu_mtrees:root_hash()} | {error, empty}.
+root_hash(#contract_tree{}) ->
+    {error, empty}.
```
Comment (lucafavatella) [inline apps/aecontract/src/aect_state_tree.erl]: Addition of TODO comment here welcome.

Comment (lucafavatella) [reply apps/aecontract/src/aect_state_tree.erl]: Ah it is empty anyway so there is not way that can be forgotten. (Other functions have PLACEHOLDER for remembering.)

**Comment by lucafavatella** [comment]
I see this PR 6ca45347 is based directly on master ca11409e , so the two CI builds (push and pr) are actually redundant:
* pr CI build still needs to start
* push CI build has not completed static_analysis and package yet

**Comment by hanssv** [comment]
The release notes already say this release is a chain altering release, not sure there is anything else to say?!

There is no API for getting aect_create_tx into the system, but if one would enter one (into attached shell maybe?) it would only affect the account tree since the `inser_contract` is just a placeholder.

**Pull request merged**